Server/DeviceManagement/DeviceManagement.Application/DataMappers/AreaDataMapper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using ModularHouse.Server.DeviceManagement.Application.Dto;
+using ModularHouse.Server.DeviceManagement.Domain.AreaAggregate;
+
+namespace ModularHouse.Server.DeviceManagement.Application.DataMappers;
+
+public static class AreaDataMapper
+{
+    public static AreaDto ToDto(this Area area)
+    {
+        return new AreaDto
+        {
+            Id = area.Id,
+            Name = area.Name,
+            Description = area.Description,
+            CreationDate = area.CreationDate,
+            CreatedByUserId = area.CreatedByUserId,
+            LastUpdatedDate = area.LastUpdatedDate,
+            LastUpdatedByUserId = area.LastUpdatedByUserId
+        };
+    }
+
+    public static IReadOnlyList<AreaDto> ToDto(this IReadOnlyList<Area> areas)
```

Comment:
You convert `List<Area>` into `List<AreaDto>`, but not `AreaListDto`, so method name `ToDto` is incorrect;
Rename it to `ToDtoList` or something similar.